### DISCUSSION TITLE
Need help on deployment on Vercel

### DISCUSSION THREAD

**Discussion by gitvivek14** [evershopcommerce/evershop#652] [Q&A]
Error: No Output Directory named "public" found after the Build completed. You can configure the Output Directory in your Project Settings

**Comment by gitvivek14** [comment]
Please Help

**Reply by gitvivek14** [reply]
Please Help!!!!!!!!!!
Anyone!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by treoden** [comment]
Hi,

Since Vercel is good for a static generated website. I am not sure it can be a good option to deploy Evershop (Require full backend nodejs and postgres DB)

Thanks

**Reply by EmanuelCG** [reply]
Can you send a picture of your directory structure?

**Reply by gitvivek14** [reply]
Sure
![image](https://github.com/user-attachments/assets/2587fd88-68a6-4ee6-9f71-8b2d8a036e67)

**Reply by gitvivek14** [reply]
Build completes on Vercel , But i am Confused what to set my Output Directory ?

**Reply by gitvivek14** [reply]
Please Help

**Reply by EmanuelCG** [reply]
You can run the npm run build command and check what errors are being generated when building, send me a screenshot.

**Reply by gitvivek14** [reply]
Do i need to include .evershop directory , while deployment , or it should be generated on npm run build

**Reply by EmanuelCG** [reply]
You do not need to include the .evershop directory.

**Reply by gitvivek14** [reply]
It is deployed , But showing me error 404

**Reply by EmanuelCG** [reply]
You can check the vercel logs, share a screenshot
